src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -323,25 +324,49 @@ where
                     .await
                     .map_err(PsqlError::ParseError)?
             } else {
-                sql.split(&[' ', ',', ';'])
+                let generic_params: Vec<&str> = sql
```

Comment:
Provide a parsing example as comment here?

Reply:
Actually I mean examples of each key processing step. So we can easily know how they are processed.